tests/_data/acceptanceDump.sql

```diff
@@ -385,7 +385,9 @@ INSERT INTO `mp_mailpoet_settings` (`id`, `name`, `value`, `created_at`, `update
 (22,	'analytics',	'a:1:{s:7:\"enabled\";s:0:\"\";}',	'2017-10-30 00:58:13',	'2017-10-30 00:58:13'),
 (23,	'premium',	'a:1:{s:11:\"premium_key\";s:0:\"\";}',	'2017-10-30 00:58:13',	'2017-10-30 00:58:13'),
 (24,	'user_seen_editor_tutorial1',	'1',	'2017-10-30 00:58:13',	'2017-10-30 00:58:13'),
-(25,	'display_nps_poll',	'0',	'2018-12-13 14:20:00',	'2018-12-13 14:20:00');
+(25,	'display_nps_poll',	'0',	'2018-12-13 14:20:00',	'2018-12-13 14:20:00'),
+(26,	'woocommerce', 'a:1:{s:30:"accept_cookie_revenue_tracking";a:2:{s:7:"enabled";s:1:"1";s:3:"set";s:1:"1";}}',  '2019-07-17 12:00:00',  '2019-07-17 12:00:00');
```

Comment:
I don't think we should do this any more. We want to get rid of this file eventually and don't add anything new to it if possible.